Sprint 2.md

```diff
@@ -0,0 +1,216 @@
+
+## Backend Unit Tests
+
+List of Backend Unit tests for Sprint 2 : 
+
+- **Account Management:** Tests for creating, updating, retrieving, and deleting user accounts.
+- **Group Management:** Tests for creating, updating, retrieving, and deleting groups.
+- **Channel Management:** Tests for creating, updating, retrieving, and deleting channels.
+- **Message Management:** Tests for creating, updating, retrieving, and deleting messages.
+
+These unit tests are implemented using the Go testing framework and are located in the `backend/tests/api/v1/` directory of the repository. To run the tests, navigate to the `backend` directory and execute the following command:
+
+
+
+
+# Backend API Documentation
```

Comment:
Add a note that we auto-generate swagger ui documentation for the project which is hosted by the server.